kits/multisig/contracts/MultisigKit.sol

```diff
@@ -28,9 +28,11 @@ contract MultisigKit is BetaKitBase {
     }
 
     function newInstance(string name, address[] signers, uint256 neededSignatures) external {
-        require(signers.length > 0);
+        require(signers.length > 0 && neededSignatures > 0);
+        require(neededSignatures <= signers.length);
+        // We can avoid safemath checks here as it's very unlikely a user will pass in enough
+        // signers to cause this to overflow
         uint256 neededSignaturesE18 = neededSignatures * 10 ** 18;
-        require(neededSignaturesE18 >= signers.length);
```

Comment:
It's probably nicer this way, but not so sure this is a fix:
- `neededSignatures > 0` was implicitly checked with `neededSignaturesE18 >= signers.length`
- `neededSignatures <= signers.length` will be checked on Voting app initialization, unless `signers.length` is bigger than `10^18`, which is assumed not to happen (and I guess that's why we are removing `neededSignaturesE18 >= signers.length`)
- I would remove `neededSignaturesE18` variable, or at least move it below to the only place it is used.

Reply:
> neededSignatures <= signers.length will be checked on Voting app initialization,

But the user will lose almost 90% (made up, but probably in that range) of their gas if we wait until then to do this check.

> I would remove neededSignaturesE18 variable, or at least move it below to the only place it is used.

Sounds good!

The reason I originally thought this was a bug was because `neededSignaturesE18 >= signers.length` looks like an unreasonable check (any `neededSignatures > 1` would pass).

Reply:
> But the user will lose almost 90% (made up, but probably in that range) of their gas if we wait until then to do this check.
Good point. Let's leave it then. Although I would always prioritize low gas consumption for successful executions than for failures.
> any neededSignatures > 0 would pass
Well, yes, as long as `signers.length` is lower than 10^18, which is a good assumption, haha.